### DISCUSSION TITLE
Support for screenshot like XSS hunter

### DISCUSSION THREAD

**Discussion by hackncode** [projectdiscovery/interactsh#318] [Ideas]
https://xsshunter.com/features

Using HTML5 canvas API for self hosted instances. 
If possible, for shared public instance.

**Comment by geeknik** [comment]
Please allow us to toggle this on and off because some programs don't allow the taking of screenshots.

**Comment by Mzack9999** [comment]
Implemented via #53

The client side code injected via the scripting engine should use `https://github.com/niklasvh/html2canvas` and post the result back to the interactsh self-hosted instance. XSSHunter implements a fully working payload at `https://your_handle.xss.ht`.
